modules/localstack/localstack.go

```diff
@@ -0,0 +1,141 @@
+package localstack
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/internal/testcontainersdocker"
+	"github.com/testcontainers/testcontainers-go/wait"
+	"golang.org/x/mod/semver"
+)
+
+const defaultPort = 4566
+const defaultRegion = "us-east-1"
+const defaultVersion = "0.11.2"
```

Comment:
Hey! This is a pretty old LocalStack version. It would be awesome to upgrade this to something post v1.0. we are releasing 1.4 next week :)

Reply:
@thrau I'm bumping it on https://github.com/testcontainers/testcontainers-go/pull/800/commits/bb4e2a87a165331588948c2026393f0065a976a8 to 1.3.1. Thanks for the suggestion!

Reply:
We updated to 1.4.0!